### PR TITLE
Migrate cloudbuild connection to mmv1

### PR DESCRIPTION
**Pull request by trodge**
Migration of cloudbuild connection and repository.

fixes https://github.com/hashicorp/terraform-provider-google/issues/16162

https://github.com/GoogleCloudPlatform/magic-modules/pull/9631 for repository only

**Release Note Template for Downstream PRs (will be copied)**

```release-note:note
cloudbuildv2: changed underlying actuation engine for `google_cloudbuildv2_connection`, there should be no user-facing impact
```

### CODE REVIEW

**Comment by trodge** [comment]
I am getting the same error with `TestAccCloudbuildv2Connection_GlePrivConnection` when I run it on the main branch.

**Comment by trodge** [comment]
These tests also fail in nightly runs: https://hashicorp.teamcity.com/test/-7589390189395444125?currentProjectId=TerraformProviders_Google_NightlyTests&expandTestHistoryChartSection=true

File: mmv1/products/cloudbuildv2/Connection.yaml [hunk 1]
```diff
@@ -40,8 +50,174 @@ examples:
...
+        description: SecretManager resource containing the webhook secret of the GitHub App, formatted as `projects/*/secrets/*/versions/*`.
+      - !ruby/object:Api::Type::Integer
+        name: appInstallationId
+        description: ID of the installation of the GitHub App.
+      - !ruby/object:Api::Type::NestedObject
+        name: serviceDirectoryConfig
+        description: Configuration for using Service Directory to privately connect to a GitHub Enterprise server. This should only be set if the GitHub Enterprise server is hosted on-premises and not reachable by public internet. If this field is left empty, calls to the GitHub Enterprise server will be made over the public internet.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: service
```
Comment (slevenick) [inline mmv1/products/cloudbuildv2/Connection.yaml:127]: Lost DSF here

**Review by slevenick — APPROVED** [review]
Can you change the changelog note to be similar to:
cloudbuildv2: changed underlying actuation engine for google_cloudbuildv2_repository, there should be no user-facing impact (https://github.com/hashicorp/terraform-provider-google/pull/16969)

**Pull request merged**